app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -139,3 +139,39 @@
     expect($serviceMonitoringTarget->teams())->exists()->toBeTrue();
     expect($serviceMonitoringTarget->users())->exists()->toBeTrue();
 });
+
+test('create service monitoring with validate the domain url', function () {
+    asSuperAdmin();
+
+    $validUrls = [
+        'http://example.com',
```

Comment:
I would test and HTTP, HTTPS, and no protocol (`example.com`) should all be valid.